File: backend/shopping-application/src/main/java/com/shopping/shoppingapplication/models/SizeVariant.java
package com.shopping.shoppingapplication.models;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class SizeVariant {

    private Size size;
    private int quantity;
}

File: frontend/E-commerce-Complete-Flutter-UI/lib/models/size_variant.dart
class SizeVariant {
  String? size;
  int? quantity;

  SizeVariant({this.size, this.quantity});

  SizeVariant.fromJson(Map<String, dynamic> json) {
    size = json['size'];
    quantity = json['quantity'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['size'] = this.size;
    data['quantity'] = this.quantity;
    return data;
  }
}